mbtiles/mp.py

```diff
@@ -0,0 +1,52 @@
+# multiprocessing Pool implementation
+
+from multiprocessing import Pool
+import warnings
+
+from mbtiles.compat import zip_longest
+from mbtiles.worker import init_worker, process_tile
+
+BATCH_SIZE = 100
+
+
+class MbtilesDeprecationWarning(FutureWarning):
```

Comment:
I'm following https://www.python.org/dev/peps/pep-0565/#additional-use-case-for-futurewarning.

> For library and framework authors that want to ensure their API compatibility warnings are more reliably seen by their users, the recommendation is to use a custom warning class that derives from DeprecationWarning in Python 3.7+, and from FutureWarning in earlier versions.

Since we're still supporting older versions in 1.5.0, I'm going with FutureWarning.